internal/command/views/json/query.go

```diff
@@ -34,15 +36,17 @@ func NewQueryStart(addr addrs.AbsResourceInstance, input_config cty.Value) Query
 	}
 }
 
-func NewQueryResult(addr addrs.AbsResourceInstance, value cty.Value) QueryResult {
-	return QueryResult{
-		Address:        addr.String(),
+func NewQueryResult(listAddr addrs.AbsResourceInstance, value cty.Value, generated *genconfig.Resource) QueryResult {
+	result := QueryResult{
+		Address:        generated.Addr.String(),
```

Comment:
The address was a reference to the list block that produced this result. We should introduce another field if we want to expose the generated reference as well. But do we need that?

Reply:
👍  Perhaps we do not. I will leave it for now